.gitlab-ci.yml

```diff
@@ -37,6 +39,48 @@ unit_tests_julia1.9:
   tags:
     - cpuonly
 
+generate_integration_tests:
+  image: julia:1.9
+  stage: generate_integration_test
+  script:
+    # extract package name
+    - export CI_DEPENDENCY_NAME=$(cat $CI_PROJECT_DIR/Project.toml | grep name | awk '{ print $3 }' | tr -d '"')
+    - echo "CI_DEPENDENCY_NAME -> $CI_DEPENDENCY_NAME"
+    - apt update && apt install -y git
+    # TODO: use upstream repo
+    #- git clone --depth 1 -b dev https://github.com/QEDjl-project/QED.jl.git /QEDjl
+    - git clone --depth 1 -b integrationTests https://github.com/SimeonEhrig/QED.jl.git /QEDjl
```

Comment:
does this clone the right repo?

Reply:
No, this needs to be fixed. I changed it.